Sources/Plasma/FeatureLib/pfDXPipeline/plDXPipeline.h

```diff
@@ -43,6 +43,7 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #define _plDX9Pipeline_h
 
 #include <vector>
+#include <d3d9.h>
```

Comment:
Glancing through the members, it looks like we could get away with using forward declarations instead of bringing in all of Direct3D9 in this header.

Reply:
`plDXPipeline.h` itself doesn't need this, but `plDXSettings.h` does (and potentially other includes do too). I can put it in `plDXSettings.h` if we want (which still ends up bringing it in to this header)

Reply:
Yeah, I think it's better to put includes into the files where the definitions are actually needed.